### PR TITLE
ENT-9184: Added ubuntu 22 x86/arm and debian 11 arm platforms 

### PR DESCRIPTION
**Pull request by craigcomstock**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -201,3 +201,11 @@ CFEngine 3.15.3 (Enterprise) was successfully installed on 'localhost'
 
 Feel free to open pull requests to expand this documentation, add features or fix problems.
 You can also pick up an existing task or file an issue in [our bug tracker](https://tracker.mender.io/issues/?filter=11711).
+
+## Development
+
+To install `cf-remote` so that it reflects any changes in this source directory use:
```
Comment (larsewi) [inline README.md:207]: Does this do the same as `pip install .`?

**Review by larsewi — APPROVED** [review]
ACK

**Review by larsewi — APPROVED** [review]
REACK

File: README.md (hunk 1 above)
Comment (craigcomstock) [reply README.md:207]: Ah, that would be `pip install -e .` I think. Which may have the advantage of letting pip know you installed it. I'll change it. Though in testing I see that using the "old way" also seems recognized by pip.

```-e, --editable <path/url>
Install a project in editable mode (i.e. setuptools “develop mode”) from a local project path or a VCS url.```

Comment (vpodzime) [inline README.md:207]: Actually, I'd prefer telling people to use `export PYTHONPATH=.` rather the messing up with their installation of `cf-remote`.

Comment (craigcomstock) [reply README.md:207]: @vpodzime I'm not familiar with that. That would pre-pend the current directory while in effect and so `cf-remote` would load modules from there? `unset PYTHONPATH` would then remove that and installed `cf-remote` would work as usual?

Comment (vpodzime) [reply README.md:207]: Yes, exactly. with `PYTHONPATH=.` things like `from cf_remote import ssh` would import `./cf_remote/ssh.py`.

**Pull request merged**